reference/promotions.v3.yml

```
@@ -195,6 +195,86 @@ paths:
         '204':
           description: The deletion was successful or the specified resource does not exist.
           content: {}
+
+  '/promotions/{promotion_id}/codegen':
+    post:
+      tags:
+        - Coupon Codes (Bulk)
+      summary: Generate Bulk Coupon Codes
+      description: |-
+        Generate a batch of coupon codes for a particular bulk coupon promotion.
+        
+        **Note:**
+        * batch_size is limited to 250 codes per request for first version. If batch_size is not an integer or larger than 250, it will return a 400 error code.
+        * The default rate limit for this endpoint is 40 concurrent requests.
```

should we start with small number first ?
```suggestion
        * The default rate limit for this endpoint is 10 concurrent requests.
```

---

Maybe lets just not expose what our concurrent limits are? Since these are dynamic, probably not very useful to add a hard number in the docs